components/actionbutton/stories/template.js

```diff
@@ -69,7 +69,7 @@ export const Template = ({
 	const { updateArgs } = context;
 	return html`
 		<button
-			aria-label=${ifDefined(label)}
+			aria-label=${ifDefined(hideLabel ? label : undefined)}
```

Comment:
This is the same why `Button` handles the `aria-labels`

Reply:
Is this preferable to wrapping the label and value in the condition and omitting altogether if the condition is false?

Reply:
It is - this is the way lit-html manages attributes